pkg/download/dependency_resolution/resolver/ahocorasick_dep_resolver.go

```diff
@@ -62,6 +63,14 @@ func AhocorasickResolver(configsById map[string]config.Config) ahocorasickResolv
 	}
 }
 
+func buildDict(ids []string) [][]rune {
```

Comment:
Yes, it builds a slice of rune slices, so an integer ID -> string "dict" - tbh I went with the naming of the library. 
It could be something like this:
```suggestion
func convertStringIDsToRuneSlices(ids []string) [][]rune {
```
Not sure if that actually helps readability though... Do you have a name idea?

Reply:
renamed and documented